lib/probes/mongodb-core.js

```diff
@@ -255,6 +255,8 @@ const dataMakers = [
       data.Group_Reduce = cmd.group.$reduce.toString()
     } else if (typeof cmd.group.$reduce === 'object') {
       data.Group_Reduce = cmd.group.$reduce.code.toString()
+    } else {
```

Comment:
```
 if (typeof cmd.group.$reduce === 'function') {
      data.Group_Reduce = cmd.group.$reduce.toString()	    
    } 
```
this is now also captured in the else clause, it seems that the test for 'object' is sufficient to cover the 2 cases.